cmd/avrogen/main.go

```diff
@@ -23,15 +23,15 @@ type config struct {
 }
 
 func main() {
-	os.Exit(realMain(os.Args, os.Stderr))
+	os.Exit(realMain(os.Args, os.Stderr, os.Stdout))
 }
 
-func realMain(args []string, out io.Writer) int {
+func realMain(args []string, out, dumpout io.Writer) int {
 	var cfg config
 	flgs := flag.NewFlagSet("avrogen", flag.ExitOnError)
 	flgs.SetOutput(out)
 	flgs.StringVar(&cfg.Pkg, "pkg", "", "The package name of the output file.")
-	flgs.StringVar(&cfg.Out, "o", "", "The output file path.")
+	flgs.StringVar(&cfg.Out, "o", "", "The output file path (dump to stdout if not provided).")
```

Comment:
Perhaps this is clearer
```suggestion
	flgs.StringVar(&cfg.Out, "o", "", "The output file path to write to instead of stdout.")
```